worker_types/deepspeech-win/userdata

```diff
@@ -76,12 +64,6 @@ $ace = New-Object System.Security.AccessControl.FileSystemAccessRule("Everyone",
 $acl.AddAccessRule($ace)
 Set-Acl "C:\builds" $acl
 
-# download tooltool
-$client.DownloadFile("https://raw.githubusercontent.com/mozilla/build-tooltool/master/tooltool.py", "C:\builds\tooltool.py")
-
-# install nssm
-Expand-ZIPFile -File "C:\nssm-2.24.zip" -Destination "C:\" -Url "http://www.nssm.cc/release/nssm-2.24.zip"
-
```

Comment:
You need to keep nssm, it is needed by generic-worker.

Reply:
done, thanks